[Studios] Form update (#942)
seqeralabs/docs: platform-cloud/docs/studios/add-studio-provided-template.md

    - **Keep the session running:** Continuous session operation until manually stopped or an error terminates it. The session continues consuming compute resources until stopped.
- **Environment variable**: Environment variables for the session. All variables from the selected compute environment are automatically inherited and displayed. Additional session-specific variables can be added. Session-level variables take precedence — to override an inherited variable, define the same key with a different value.
- **Resource labels**: Any [resource label](../labels/overview) already defined for the compute environment is added by default. Additional custom resource labels can be added or removed as needed.

### Mount data

Mount data repositories to make them accessible in your session:

1. Select **Mount data** to open the data selection modal.
1. Choose the data repositories to mount.
1. Select **Mount data** to confirm.

> Mounted repositories are accessible at `/workspace/data/<DATA_REPOSITORY>` using the [Fusion file system](https://docs.seqera.io/fusion). Data doesn't need to match the compute environment region, though cross-region access may increase costs or cause errors.

Reviewer: ```suggestion
Once the Studio session is running, mounted data are accessible at `/workspace/data/<DATA_LINK_NAME>` using the [Fusion file system](https://docs.seqera.io/fusion). Data doesn't need to match the compute environment region, though cross-region data transfer (ingress and egress) may increase costs.
```

---
title: "Seqera-provided container template"
description: "Add a Studio in Platform."
date created: "2025-09-04"
last updated: "2025-11-14"
tags: [template, seqera-template, seqera-template-studio, session, studios]
---

:::info[**Prerequisites**]
You will need the following to get started:

- Valid credentials for accessing cloud storage resources

> - **Maintain** role permissions (minimum)

Reviewer 2: ```suggestion
- **Maintain** role permissions or above
```

- A compute environment with sufficient resources (scale based on data volume)
- [Data Explorer](../data/data-explorer) enabled
:::

Configure the following fields:

- **Container template**: The template for the container. Select a provided container template.
- **Studio name**: The name for the Studio.
- **Description** (optional): A description for the Studio.
- **Install Conda packages**: A list of conda packages to include with the Studio. For more information on package syntax, see [conda package syntax][conda-syntax].
  :::note

>   The workspace Admin needs to set a target repository per workspace, in **Settings > Studios > Container repository**. If no repository configuration is specified, the build will fail. Each workspace needs to have credentials available in the workspace to push to the specified repository.

Author: ```suggestion
  The workspace Admin needs to set a target repository per workspace, in **Settings > Studios > Container repository**. If no repository configuration is specified, the build will fail. Each workspace must have credentials available to push to the specified repository.
```
